examples/nlp/scripts/get_wkt2.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
This makes some docs obsolete: https://nvidia.github.io/NeMo/nlp/pretraining.html